package.json

```diff
@@ -31,11 +31,10 @@
     "@newrelic/test-utilities": "^4.1.2",
     "apollo-server": "^2.18.2",
     "eslint": "^7.8.1",
-    "newrelic": "^6.13.0",
+    "newrelic": "^7.3.1",
     "tap": "^14.10.8"
   },
   "peerDependencies": {
-    "newrelic": ">=6.13.0"
-  },
-  "dependencies": {}
+    "newrelic": ">=7.3.1"
```

Comment:
We currently are preferring this to be a peer dependency. We only allow for one version of the agent to be loaded at a time and will use (regardless) what the application is currently setup to use. While we do have a safety mechanism in place for if this is set as a standard dependency, it will log an unnecessary warning for trying to load the agent more than once.

I would revert this part of the change.

Reply:
got it! will do